src/microbe_stage/Microbe.Contact.cs

```diff
@@ -383,16 +424,124 @@ public bool CanEngulf(Microbe target)
             return false;
         }
 
+        // Limit amount of things that can be engulfed at once
+        if (IngestedSizeCount > EngulfSize || IngestedSizeCount + target.EngulfSize > EngulfSize)
+            return false;
+
+        // Too many things attempted to be engulfed at once
```

Comment:
So this means that currently being pulled into the cell? This comment is a bit vague, so I'd like that bit of clarification here.

Reply:
Yes, will reword this.